docs/source/content/api.md

```diff
@@ -137,6 +141,7 @@ environment to ensure reproducible samples.
 ```  
 
 ## Checking API-Conformity
+<a name="API-checker"></a>
```

Comment:
What does this do?

Reply:
I was wanting to link the section above to the API-conformity above. I don't think that I tested it admittedly

Reply:
You can check the vercel preview to see how it appears (or run locally).